seedVault/encryptionUtils.ts

```diff
@@ -0,0 +1,106 @@
+import { mnemonicToEntropy, wordlists } from 'bip39';
+
+interface EncryptMnemonicArgs {
+  password: string;
+  seed: string;
+  mnemonicEncryptionHandler: (seed: string, key: string) => Promise<Buffer>;
+}
+
+export async function encryptMnemonicWithCallback(cb: EncryptMnemonicArgs) {
```

Comment:
this naming is a bit confusing because we don't actually do a callback 🤔 Maybe it should be something like `encryptMnemonicWithHandler`

Reply:
Yes, that is a bit confusing. Was this originally meant to have a callback?